cmd/vic-machine-server/Dockerfile

```diff
@@ -0,0 +1,12 @@
+# Building:
+# from vic root directory
+# docker build --no-cache -t vic-machine-server -f cmd/vic-machine-server/Dockerfile .
+# docker tag vic-test gcr.io/eminent-nation-87317/vic-machine-server:1.x
+# gcloud auth login
+# gcloud docker -- push gcr.io/eminent-nation-87317/vic-machine-server:1.x
+
+FROM golang:1.8
+
+COPY bin/vic-machine-server /vmware/
+
+ENTRYPOINT /vmware/vic-machine-server
```

Comment:
I agree with the ENVs but on the appliance, vic-machine-server can't listen on 80/443 since there are already services running there. Do you still want me to put the EXPOSEs?

Reply:
Added EXPOSE since it doesn't hurt anything and can be useful for testing